ports/geckolib/gecko_bindings/bindings.rs

```diff
@@ -156,6 +158,26 @@ pub type RawGeckoDocument = nsIDocument;
 pub enum ServoNodeData { }
 pub enum ServoComputedValues { }
 pub enum RawServoStyleSheet { }
+#[repr(C)]
+#[derive(Debug)]
+pub struct ServoComputedValuesStrong {
+    pub mPtr: *mut ServoComputedValues,
```

Comment:
For some reason bindgen doesn't make this private. Perhaps it can't handle macros?